instrumentation/google.golang.org/grpc/otelgrpc/go.mod

```diff
@@ -7,22 +7,23 @@ require (
 	go.opentelemetry.io/otel v1.11.2
 	go.opentelemetry.io/otel/metric v0.34.0
 	go.opentelemetry.io/otel/trace v1.11.2
-	google.golang.org/grpc v1.51.0
+	google.golang.org/grpc v1.53.0-dev.0.20230103192020-f2fbb0e07ebf
```

Comment:
This is not a stable and tested release made by the project.

Reply:
Yes, I should have noted that in the PR description — this is a draft PR precisely because I’m waiting for the grpc release, I just wanted to see what the impact was (including CI).